### PR TITLE
Add pages.markdown.extra_escape_fences feature

### PR DESCRIPTION
**Pull request by Flamenco**

### CODE REVIEW

**Comment by Flamenco** [comment]
Sample Issue

```html
<p>
    This is a <HR /> 
    <FOO>test</FOO>
</p>
```

<div markdown='1'>
```html
<p>
    This is a <HR /> 
    <FOO>test</FOO>
</p>
``` 
</div>

**Comment by Flamenco** [comment]
The fence in the diff erroneously:
- Has case transformed to lower
- Looses whitespace formatting
- Creates closing tags

**Comment by Flamenco** [comment]
before:
before
after:
after

File: system/blueprints/config/system.yaml [hunk 1]
```diff
@@ -430,6 +430,16 @@ form:
                         0: PLUGIN_ADMIN.NO
                     validate:
                         type: bool
+                pages.markdown.extra_escape_fences:
+                    type: toggle
+                    label: Escape HTML elements in markdown extra fences
```
Comment (rhukster) [inline system/blueprints/config/system.yaml]: This label should also be a lang string, and should be added to admin's `en.yaml` lang file.

File: system/src/Grav/Common/Page/ParsedownExtraUtil.php [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<?php
```
Comment (rhukster) [inline system/src/Grav/Common/Page/ParsedownExtraUtil.php]: This file should be in `Grav/Common/Markdown/` not in `Grav/Common/Pages/`

**Comment by Flamenco** [comment]
@rhukster I think this is ready to go.  The i8n strings made it into the admin plugin. 
https://github.com/getgrav/grav-plugin-admin/pull/1331/files

**Comment by Flamenco** [comment]
I made another PR.  Closing this one.

**Pull request closed without merging**